Formula/fibjs.rb

```diff
@@ -1,9 +1,9 @@
 class Fibjs < Formula
   desc "JavaScript on Fiber"
   homepage "http://fibjs.org/"
-  url "https://github.com/fibjs/fibjs.git",
-      :tag => "v0.5.0",
-      :revision => "92a0d14ee241fffd29603963cafb1c1bc9a0dcf8"
+  url "https://github.com/fibjs/fibjs/releases/download/v0.6.1/fullsrc.zip"
+  version "0.6.1"
```

Comment:
Does that outweigh the advantages of a proper source archive that doesn't require all the submodules?